RealmSwift/Realm.swift

```diff
@@ -300,7 +300,7 @@ public final class Realm {
     :nodoc:
     */
     @discardableResult
-    public func createDynamicObject(ofType typeName: String, populatedWith value: Any = [:], update: Bool = false) -> DynamicObject {
+    public func dynamicCreate(_ typeName: String, value: Any = [:], update: Bool = false) -> DynamicObject {
```

Comment:
It might be nice for this to share a prefix with `create`. For example, `create(typeNamed:value:update:)`.